### DISCUSSION TITLE
books/less_books

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#61] [Announcements]
# books/less_books

本文为您推荐计算机科学领域的精选学习资源，涵盖计算机思维、C语言、算法、C++、网络编程和操作系统，旨在帮助您系统学习计算机科学知识。

https://csguide.cn/books/less_books.html

**Comment by wozien** [comment]
大佬牛逼

**Comment by Kiemss** [comment]
感谢大佬写的指南，非常有用！
